driver/driver.go

```diff
@@ -53,10 +53,11 @@ func FromSource(source models.Source) (Driver, error) {
 			Credentials:      creds,
 			S3ForcePathStyle: aws.Bool(true),
 			MaxRetries:       aws.Int(maxRetries),
+			DisableSSL:       aws.Bool(source.DisableSSL),
 		}
 
 		if len(source.Endpoint) != 0 {
-			endpoint := fmt.Sprintf("https://%s", source.Endpoint)
+			endpoint := fmt.Sprintf("%s", source.Endpoint)
```

Comment:
To my understanding the aws library adds a scheme if there is no scheme added to the endpoint url, the DisableSSL option adds either `https` if false and `http` if true.